utils/constants.js

```diff
@@ -1,20 +1,32 @@
+const path = require('path')
+
 const STATUS = {
-	5: 'Running',
-	14: 'Wrong answer',
-	16: 'Success',
-	RUNNING: 5,
+    5: 'Running',
+    14: 'Wrong answer',
+    16: 'Success',
+    RUNNING: 5,
 }
 
 const TEST_STATUS = {
-	'Accepted': '✅',
-	ACCEPTED: '✅',
-	'Wrong Answer': '❌',
-	WRONG_ANSWER: '❌',
-	'not checked': '❔',
-	NOT_CHECKED: '❔',
+    'Accepted': '✅',
+    ACCEPTED: '✅',
+    'Wrong Answer': '❌',
+    WRONG_ANSWER: '❌',
+    'not checked': '❔',
+    NOT_CHECKED: '❔',
+    RUN_TIME_ERROR: '⚠️',
```

Comment:
Haha, speaking from experience I take it? 😉 😉 😉

Reply:
Autoformat...😥

Reply:
Ah, I get what you mean now. Yes apparently run time error was a thing when they removed a problem which I was testing on...